klog.go

```diff
@@ -830,11 +830,13 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		// First, make sure we see the trace for the current goroutine on standard error.
 		// If -logtostderr has been specified, the loop below will do that anyway
 		// as the first stack in the full dump.
+		trace := stacks(true)
 		if !l.toStderr {
 			os.Stderr.Write(stacks(false))
+		} else {
+			os.Stderr.Write(trace)
```

Comment:
So this prints:
- with `-logtostderr`: trace for all goroutines
- without `-logtostderr`: trace only for the current goroutine

Right?

I am not sure if we need/want this different behaviour? Any input? Tagging random people for some more opinions: @dims @neolit123 @detiber @munnerz @pohly 

Also: Either I don't misread/misunderstand the [above comment](https://github.com/kubernetes/klog/pull/79/files#diff-fb778b051a785bbb2eea40d6a00a3a21L830-L832), but do me it doesn't make sense. Can you reword it, to capture the current intent?

Reply:
that behavior is fascinating strange (chalk that one up to another oddity of glog).

I agree the comment needs to be reworded -- at the very least, we're changing the behavior slightly so that we're no longer relying on the loop.